src/features/annotation/components/task-detail/annotation-form-panel/annotation-form.tsx

```diff
@@ -17,9 +17,9 @@ import {
   isSexEnabled,
 } from './validation/annotation-form-schema';
 import { cn } from '@/shared/core/utils';
-import { Textarea } from '@/ui/textarea';
-import { Button } from '@/ui/button';
-import { Toggle } from '@/ui/toggle';
+import { Textarea } from '@/shared/ui/textarea';
+import { Button } from '@/shared/ui/button';
+import { Toggle } from '@/shared/ui/toggle';
```

Comment:
@/ui/... is already an alias for @/shared/ui, so you can change these imports back to what they were. I created the alias to make the imports less wordy.